README.rst

```diff
@@ -165,7 +165,7 @@ Issues
 ------
 
 Please use `Github's issue
-tracker <https://github.com/dssg/johnson-county-ddj-public/issues>`__
+tracker <https://github.com/dssg/johnson-county-ddj-public/issues>`
```

Comment:
Do you mean to change the way this is linkified?  The ReST syntax here is horrible, but the original actually means to make a html link a la the markdown `[GitHub's issue tracker](link)`.